framework-core/src/main/java/org/daisy/pipeline/job/Index.java

```diff
@@ -0,0 +1,27 @@
+package org.daisy.pipeline.job;
+
+public class Index {
+        private String index;
+
+        /**
+         * Creates an index based on a string
+         * @param index
+         */
+        public Index(String index) {
+                this.index = index;
+        }
+
+        public Index stripPrefix() {
+                int idx=this.index.indexOf('/');
```

Comment:
this does not check that the `/` is actually at the end of the path. Can't this cause issues? Can't there be a `/` in the middle of the path?